### PR TITLE
Separates refresh count and fromResumeCallback url params.

### PR DESCRIPTION
**Pull request by glevitzky**
The 'rc' url parameter has been used with two different semantics, even though only one is canonical. This PR addresses the issue by using `rc` only for refresh count, and creating a new parameter, `frc`, for fromResumeCallback.

### CODE REVIEW

File: extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js [hunk 1]
```diff
@@ -526,7 +523,8 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
           (this.jsonTargeting_ &&
             this.jsonTargeting_['categoryExclusions']) || null),
       'ifi': this.ifi_,
-      rc,
+      'rc': this.refreshCount_ ? this.refreshCount_ : null,
```
Comment (keithwrightbos) [inline extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js]: 'rc': this.refreshCount_ || null

File: extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js [hunk 2]
```diff
@@ -526,7 +523,8 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
...
+      'frc': this.fromResumeCallback ? '1' : null,
```
Comment (keithwrightbos) [inline extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js]: 'frc': Number(fromResumeCallback) || null

File: extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js (hunk 1 above)
Comment (glevitzky) [reply extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js]: Done.

File: extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js (hunk 2 above)
Comment (glevitzky) [reply extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js]: Done.

**Pull request merged**